net/tcp/tcp_recvwindow.c

```diff
@@ -164,17 +165,24 @@ uint16_t tcp_get_recvwindow(FAR struct net_driver_s *dev)
 
       recvwndo = (uint16_t)rwnd;
     }
+  else if (IOB_QEMPTY(&conn->readahead))
+    {
+      /* Advertise maximum segment size for window edge if here is no
```

Comment:
-/* Advertise maximum segment size for window edge if here is no
+/* Advertise maximum segment size for window edge if there are no